src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+
+body {
+  margin: 0;
+}
+
```

Comment:
It would be right to create _reset.scss to throw values there

Reply:
Thank you. I fixed this.